src/main/java/org/apache/rocketmq/dashboard/config/RMQConfigure.java

```diff
@@ -77,6 +81,18 @@ public String getNamesrvAddr() {
         return namesrvAddr;
     }
 
+    public String getNamesrvAddrs() {
+        return namesrvAddrs;
+    }
+
+    public void setNamesrvAddrs(String namesrvAddrs) {
+        this.namesrvAddrs = namesrvAddrs;
+        List<String> nameSrvAddrList = Splitter.on(";").splitToList(this.namesrvAddrs);
```

Comment:
If multiple namesrv services are deployed in a rocketmq cluster, namesrv addresses are connected using ';' by default. So there is a problem If ';' are used to divide multiple rocketmq clusters in here, it cannot distinguish between two namesrv addresses belonging to the same rocketmq cluster or to different rocketmq clusters.

Reply:
IMO, we don't need to distinguish whether namesrvs belong to the same cluster. Because if two namesrv points to the same cluster, we only need to choose one of them to work, right?

Reply:
@zhangjidi2016  However, we can also use `@` to distinguish the namesrv of different clusters, just like this:

`rocketmq.config.namesrvAddrs=localhost1:9876;localhost2:9876@localhost3:9876;localhost4:9876`

![image](https://user-images.githubusercontent.com/46882838/148031172-8294eae5-10a6-4960-83c1-e9ef4e00ba7d.png)

**I will fix it. :-D**

Reply:
You can use application.yml instead of the application.properties configuration file, and namesrv addresses for multiple clusters can be configured in this way.
![image](https://user-images.githubusercontent.com/18254437/148032460-60c748ef-1e02-4adb-88c3-950f415c71e3.png)